app/models/fraud/indicators/timezone.rb

```diff
@@ -18,6 +18,10 @@ class Timezone < ApplicationRecord
       def self.safelist
         all.pluck(:name).push(nil)
       end
+
+      def active
```

Comment:
The form checkbox helper requires a boolean-esque value -- using activated_at directly doesnt work correctly. This and the domain model has this method implemented.